recipes/pyviz/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "pyviz" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyviz/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: bcf261c81b0aa5b17f05557519dae0eccdb57c6632ab8aa1c54c958e7c8435b7
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - param >=1.8.1
+    - pyct >=0.4.4
+    - setuptools >=30.3.0
+  run:
+    - bokeh >=1.0.4,<1.1.0
+    - holoviews >=1.11.2
+    - geoviews >=1.6.2
+    - hvplot >=0.4.0
+    - datashader >=0.6.9
+    - panel >=0.4.0
+    - param >=1.8.2
+    - numpy >=1.13
+    - ipython >=5.4.0,<=7.1.1
+    - notebook >=5.5
+    - pandas >=0.23.4,<0.25.0
+    - xarray >=0.11.0,<0.12.0
+    - dask >=1.0
+    - networkx
+    - scikit-image
+    - cartopy
+    - geopandas
+    - streamz >=0.3.0
+    - selenium
+    - phantomjs
+    - fastparquet >=0.2.1
+
+test:
+  imports:
+    - pyviz
+
+about:
+  home: http://pyviz.org
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'How to solve visualization problems with Python tools.'
+  description: |
+    This repository provides examples, demos, and training materials
+    documenting how to solve visualization problems using Python tools
+    that work with web browsers, including Bokeh, HoloViews, GeoViews,
+    Matplotlib, Datashader, Param and Panel.
+  doc_url: http://pyviz.org
+  dev_url: https://github.com/pyviz/pyviz
+
+extra:
+  recipe-maintainers:
+    - jbednar
+    - philippjfr
+    - jsignell
+    - jlstevens
+
```

Comment:
```suggestion

```

Not sure this will work, but hopefully this can fix the issue picked up by the linter...